prody/database/pfam.py

```diff
@@ -473,36 +475,49 @@ def fetchPfamPDBs(**kwargs):
                 raise ValueError('Please provide an integer for start or end when using query.')
 
     from ftplib import FTP
-    data = []
+    import gzip
+
+    data_file = open('pdbmap.gz','wb')
```

Comment:
I see the possibility of streaming the data here with [BytesIO](https://docs.python.org/release/3.1.3/library/io.html#binary-i-o), so everything can be done in memory and no need for memory and hard drive transmission. We can talk tomorrow on this.

Reply:
ok